nginx/config.go

```diff
@@ -205,6 +205,11 @@ http {
 			allow 127.0.0.1;
 			deny all;
 		}
+    location /nginx_status {
+      stub_status on;
+      allow 127.0.0.1;
+      deny all;
+    }
```

Comment:
@pixeleet Yes... I use VSCode. Previously I used Atom. With both, the editor can be augmented via packages / modules, etc. to run `gofmt -w` every time you save so that formatting violations are automatically fixed...

That being said, it probably wouldn't catch this _particular_ formatting issue since the issue here is actually within a "here doc" aka a long string.